src/resolve-script-name.js

```diff
@@ -0,0 +1,28 @@
+import {get} from 'lodash'
+
+export default resolveScriptName
+
+function resolveScriptName(scriptToFind, scriptsConfig) {
```

Comment:
Not really, because this function search for a property which contains given value (`scriptToFind`). Function which resolve what you mentioned is already implemented and it is a `prefix-matcher` package.